events/lifestyles/warfare_lifestyle/martial_strategy_events.txt

```diff
@@ -363,6 +363,34 @@ martial_strategy.1051 = {
 		add_prestige = medium_prestige_gain
 		add_dread = minor_dread_gain
 	}
+
+	# Warcraft
+	option = {
+		name = martial_strategy.1051.g.wc
+		trigger = {
+			has_trait = callous
+		}
+		trait = callous
+
+		set_nickname_effect = { NICKNAME = nick_the_ice_hearted }
+		add_dread = major_dread_gain
+		add_prestige = minor_prestige_gain
```

Comment:
This also was a part of my suggestion. Being callous isn't really dreadful.
```suggestion
		add_dread = minor_dread_gain
		add_prestige = medium_prestige_gain
```

Reply:
done in fd93bdf4